console/atest-ui/src/App.vue

```diff
@@ -60,6 +61,24 @@ const handleSelect = (key: string) => {
   panelName.value = key
   window.localStorage.setItem('activeMenu', key)
 }
+
+const locale = ref('')
+locale.value = localStorage.getItem('locale') != null ? localStorage.getItem('locale') as string : 'zh';
```

Comment:
You can read/write the language setting from the browser-based preference. See also https://github.com/LinuxSuRen/api-testing/blob/ba485ce9227ff3388fbe30f5931b13485cbcb767/console/atest-ui/src/views/cache.ts#L76